mscore/scoretab.cpp

```diff
@@ -42,6 +42,26 @@
 
 namespace Ms {
 
+//---------------------------------------------------------
+//   MsTabBar
+//    Implemented the ability of closing tab by middle click
+//---------------------------------------------------------
+
+MsTabBar::MsTabBar(QWidget* parent)
+   : QTabBar(parent)
+      {}
```

Comment:
I'd prefer a one-liner here.

Reply:
`MsTabBar::MsTabBar(QWidget* parent) : QTabBar(parent) {}`?

Reply:
yes

Reply:
Then I guess it can be put in header file?

Reply:
possible but not needed

Reply:
It's weird to have one-line definitions in .cpp file... At least for now

Reply:
having a null function is strange indeed. But seems needed here for the constructor and the inheritance. Maybe move to scoretab.h?

Reply:
And indeed seem better.